acceptance_tests/features/social_sign_in_internal.feature

```diff
@@ -25,17 +25,17 @@ Feature: Internal user signs in to social UI
     Given the user has an active account and is assigned a username and password (social)
     When they enter an incorrect username and password (social)
     Then the user is notified that an authentication error has occurred (social)
-
+  @skip
```

Comment:
Can we put in a comment as to why these are being skipped please?

Reply:
👍

Reply:
added a lil' doc string